manifests/h/HHDSoftware/DupeCare/1.01.00.0159/HHDSoftware.DupeCare.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: HHDSoftware.DupeCare
+PackageVersion: 1.01.00.0159
+PackageLocale: en-US
+Publisher: HHD Software Ltd.
+PackageName: DupeCare

```

```suggestion
PackageName: HHD Software DupeCare
```
Should match control panel as closely as possible